WORKSPACE.bazel

```diff
@@ -0,0 +1,76 @@
+workspace(
+    name = "universe",
+    managed_directories = {
+        "@npm": ["node_modules"],
+    },
+)
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "build_bazel_rules_nodejs",
+    sha256 = "b16a03bf63952ae436185c74a5c63bec03c010ed422e230db526af55441a02dd",
+    urls = [
+        "https://github.com/bazelbuild/rules_nodejs/releases/download/2.1.0/rules_nodejs-2.1.0.tar.gz",
+    ],
+)
+
+load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install")
+
+node_repositories(
+    node_repositories = {
+        "14.12.0-darwin_amd64": ("node-v14.12.0-darwin-x64.tar.gz", "node-v14.12.0-darwin-x64", "c91a4ea40289886799115a8a309b844975d59d457cbad2060779286f0a8ad01b"),
+        "14.12.0-linux_amd64": ("node-v14.12.0-linux-x64.tar.gz", "node-v14.12.0-linux-x64", "f430bf1d8352c18d628771e7c5f932dbc1e48cec1c8b6417a7bdc4027518f5ed"),
+    },
+    node_urls = ["https://nodejs.org/dist/v{version}/{filename}"],
+    node_version = "14.12.0",
+    package_json = ["//:package.json"],
+    yarn_repositories = {
+        "1.22.5": ("yarn-v1.22.5.tar.gz", "yarn-v1.22.5", "c664fb4692e4dfea750a37a533780834b40198c00cef4bbc5e8c14abab2ac141"),
+    },
+    yarn_urls = ["https://github.com/yarnpkg/yarn/releases/download/v{version}/{filename}"],
+    yarn_version = "1.22.5",
+)
+
+yarn_install(
+    name = "npm",
+    package_json = "//:package.json",
+    yarn_lock = "//:yarn.lock",
+)
+
+# Bazel Buildifier is written in Go and hence needs rules_go to be built.
+# See https://github.com/bazelbuild/rules_go for the up to date setup instructions.
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "b725e6497741d7fc2d55fcc29a276627d10e43fa5d0bb692692890ae30d98d00",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.24.3/rules_go-v0.24.3.tar.gz",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.24.3/rules_go-v0.24.3.tar.gz",
+    ],
+)
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+
+go_rules_dependencies()
```

Comment:
What are we using go_rules for?

Reply:
Buldifier is written in Go and it requires to be set like this, see: https://github.com/bazelbuild/buildtools/blob/e6efbf6df90bec363c3cbd564b72be6c8a309f14/buildifier/README.md